Provider/src/main/java/com/spotify/confidence/client/network/ApplyFlagsInteractor.kt

```diff
@@ -52,17 +47,9 @@ internal class ApplyFlagsInteractorImpl(
 @Serializable
 internal data class ApplyFlagsRequest(
     val flags: List<AppliedFlag>,
-    @Contextual
+    @Serializable(DateSerializer::class)
     val sendTime: Date,
     val clientSecret: String,
     val resolveToken: String,
     val sdk: Sdk
-)
-
-private val json = Json {
-    serializersModule = SerializersModule {
-        contextual(UUIDSerializer)
-        contextual(DateSerializer)
-        contextual(StructureSerializer)
```

Comment:
Do we not need these serializer?

Reply:
now they are handled in the data class declaration instead of declaring them in the Json object like here.